pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+### Pull Requests
+
+Please be sure you have read our documentation regarding contributing
+here: https://docs.pulpproject.org/index.html
+
+There you can read a Contribution Checklist and learn more about
+appropriate branches and rebasing among other things.
+
+Once you are ready to create a pull request:
```

Comment:
This information is very similar to https://docs.pulpproject.org/en/3.0/nightly/contributing/pull-request-walkthrough.html

It might be best to cut this down a bit, and link to that. Some of the information here is not included in the docs, and could be added there.

Reply:
I think we need to decide on a definitive version and then link to it so that we are not having to maintain things in multiple places.  As someone coming to the pulpproject.org website, there is nowhere that it is clear that these instructions differ, and no reason to suspect that you should use anything other than the docs listed as "current and stable".  If we are to use the nightly, then we need to update the current stable information to reflect those choices.

Reply:
Where are the other versions?